bin/brew

```diff
@@ -45,5 +85,77 @@ fi
 
 export HOMEBREW_BREW_FILE
 export HOMEBREW_RUBY_PATH
+export HOMEBREW_PREFIX
+export HOMEBREW_REPOSITORY
+export HOMEBREW_LIBRARY
+export HOMEBREW_CELLAR
+
+if [ -n "$HOMEBREW_OSX" ]
+then
+  if [ "$(xcode-\select --print-path)" = "/" ]
+  then
+    cat >&2 <<EOS
```

Comment:
How about we implement `odie` as such:
```bash
odie() {
    if [[ -t 1 ]]
    then
        echo -ne "\033[4;31mError\033[0m: " >&2
    else
        echo -n "Error: " >&2
    fi
    if [[ $# == 0 ]]
    then
        /bin/cat >&2
    else
        echo "$*" >&2
    fi
    exit 1
}
```

It supports all of features we talked about: highlight, heredoc, multilines and printing to stderr.